src/pages/faq.md

```diff
@@ -6,76 +6,58 @@ metaDescription: "Frequently Asked Questions about Qri"
 
-Dat, like git, uses a repository model. So, if a user were to create a dataset containing a certain csv file, and another user had the exact same csv file (because, say, they were both downloaded from the same URL around the same time), the files would not ‘collide’ - meaning both physical files would not deduplicate, and storage costs (and all other complications) would scale as more datasets are added to the network by other users.
+Dat, like git, uses a repository model. So, if a user were to create a dataset containing a certain csv file, and another user had the exact same csv file (because, say, they were both downloaded from the same URL around the same time), the files would not ‘collide’ — meaning both physical files would not deduplicate, and storage costs (and all other complications) would scale as more datasets are added to the network by other users.
 
 In an ideal data commons, identical datasets would collide and deduplicate to a single file on one’s hard drive. That doesn’t happen with Dat, BitTorrent or several others. The only technology that has that key property is IPFS, because of the ‘single swarm’ style of content propagation.
 
 We’re really impressed with the Dat community and the aims of the project as a whole, and eventually we’d like to be able to support exporting to Dat from qri.
 
-
-### What kinds of data does qri support?
-
-ALL kinds of data! We're rapidly building in support for every and any kind of data. Today, qri works best with CSV & JSON files - you can put all kinds of data in CSV & JSON files.
-
+### Which data formats does Qri support? How does qri support / get along with some of the more obscure dataset formats?
+ 
+Qri supports CSV, JSON, XSLX and CBOR formats. Much of our tooling and GUIs are optimized for tabular data (CSVs), but the other formats can be used with Qri CLI. 
 
 ### Who can see my data?
 
-Currently **all data on qri is public.** We’re working on support for both encrypted data and private networks. You can track the progress of encryption and private networks feel free to +1 any of those comments to let us know you want them faster.
+Currently all datasets on qri are public. By publishing a dataset, you make it discoverable on Qri.cloud. Unpublished datasets are still discoverable by other qri users, but it’s more akin to having an unlisted phone number. 
 
-
-### How much does it cost?
-
-It’s free! Really! Qri is free forever, all you need to bring is your own hard drive.
-
-
-### How much Data can you store in Qri?
-
-As much as your hard drive(s) can hold.
+We’re working on support for both encrypted data and private networks. You can track the progress of encryption and private networks in the [roadmap](https://github.com/qri-io/qri/issues/1093). Please thumbs-up or comment on features you want us to work on.
```

Comment:
@rgardaphe I edited this paragraph.

The old copy is up at (pink) [line 62](https://github.com/qri-io/website/pull/213/files#diff-6a96ffd27ee0d617fff3c15eef7bb79aL62) because of squashing.

Other changes are done. If this paragraph is good, you can merge the FAQ.